data/korath/korath missions.txt

```diff
@@ -11,6 +11,18 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+# This mission sets the "know who the korath are" variable in old saves.
+mission "000 Do I know the Korath?"
+	landing
+	source
+	to offer
+		or
+			has "event: label korath space"
+			has "FW Pug 2B: offered"
```

Comment:
The purpose of the condition is to indicate that you know the Korath, and there must be a condition with that specific purpose so that missions can trigger on whether you know the Korath. Whether this comes from an event or a mission does not affect the necessity of a condition with that purpose.